packages/venia-concept/src/index.css

```diff
@@ -7,6 +7,7 @@ html {
     --venia-font: Muli, -apple-system, BlinkMacSystemFont, sans-serif;
     --venia-grey: 246, 246, 246;
     --venia-teal: 0, 134, 138;
+    --venia-white: 255, 255, 255;
```

Comment:
It's true that this follows convention, but there's a reason we don't have variables already for black and white: they are constants. Our definition of teal may change, and all references to `venia-teal` should change at that time. But black must always be `#000` and white must always be `#fff`, and most references to those colors should not change even if we overhaul the whole design.

tl;dr:

- Remove this and don't use it
- `venia-white` and `venia-black` would be constant, so just use `white` and `black` instead
- If you want a variable for a value that happens to be white or black, make a derived variable for it